index.html

```diff
@@ -1155,20 +1155,20 @@ <h2>Constrainable Properties</h2>
       </table>
     </section>
     <section class="informative">
-      <h2>Timestamp concepts</h2>
+      <h2>Video timestamp concepts</h2>
       <p>
-        Media flowing inside media stream tracks comprise of a sequence of media frames, where
+        Video media flowing inside media stream tracks comprise of a sequence of video frames, where
         the frames are sampled from the media at instants spread out over time.
       </p>
+      <p>
+        Each video frame has a <dfn class="export">presentation timestamp</dfn>
+        which is relative to the first frame appearing on the track.
+      </p>
       <p>
         Each frame may have a <dfn class="export">capture timestamp</dfn> representing
         the instant it was captured. A source of frames can define how this timestamp is set, otherwise
```

Comment:
Is captureTime defined as the time at which the first octet was captured (as it is in RTP)? Might be helpful to clarify.

Reply:
My gripe against the RTP def is that "octet" isn't accurate with how many capturers produce images (or if there are octets at all). 
I've extended the definition now, PTAL & resolve if happy.